docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml

```diff
@@ -225,7 +225,7 @@
         </listitem>
 
         <listitem>
-          <para>Select the security Plugin for each service that requires a
+          <para>Select the security Plug-in for each service that requires a
```

Comment:
This instance is outside of the scope of this issue. Note I opened https://track.hpccsystems.com/browse/HPCC-17400 specifically for these other occurrences.